.github/workflows/build.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       working-directory: ${{ matrix.vcpkg_path }}
 
     # update cmake to 2.29.2 to work around https://github.com/microsoft/vcpkg/issues/37968   
-    - name: "[macOS] Set up cmake"
+    - name: "Set up cmake"
```

Comment:
This commit has already been merged into the 2.6 branch. Can you rebase it out of this PR?